src/private/mapbox-types.ts

```diff
@@ -120,6 +102,22 @@ export interface FeatureIndex {
     availableImages: string[],
     styleLayers: { [layerId: string]: StyleLayer },
   ): QueryResult
+  // v3.8.0 or later
+  lookupSymbolFeatures(
+    symbolFeatureIndexes: number[],
+    bucketIndex: number,
+    sourceLayerIndex: number,
+    // original definition of `filterSpec` is complicated `FilterSpecification`
+    // that I wanted to avoid to redefine.
+    // this should not matter to this library.
+    // may be `undefined` or `null` in effect
+    filterSpec: any,
+    // original definition of `filterLayerIDs` is `string[]`,
+    // but may be `undefined` or `null` in effect
+    filterLayerIDs: string[] | undefined | null,
```

Comment:
Should not accept `undefined` and `null`.
```suggestion
    filterLayerIDs: string[],
```